bundle/outputs.go

```diff
@@ -6,3 +6,17 @@ type Output struct {
 	Description string   `json:"description,omitempty" yaml:"description,omitempty"`
 	Path        string   `json:"path" yaml:"path"`
 }
+
+// AppliesTo satisfies the ActionApplicable interface
+// by determining whether or not the Output applies to the provided action
+func (o *Output) AppliesTo(action string) bool {
```

Comment:
This method seems to be a duplicate of `func (p *Parameter) AppliesTo`, should we factorize it ?